benchmark/main.py

```diff
@@ -8,6 +8,7 @@
 from clickhouse_driver import Client
 
 from asynch import connect
+from conftest import CONNECTION_DSN
```

Comment:
It seems to me that here I violated the module's area of responsibility by rigidly linking "benchmark" and "conftest" (although it is in the root directory). Of course, this was done with the best of intentions: to avoid code duplication, but in theory this could cause problems in the future.

If you also think this is a problem: tell me - I'll fix it.